src/TwigComponent/tests/Unit/TwigPreLexerTest.php

```diff
@@ -216,5 +216,31 @@ public function getLexTests(): iterable
             '{% verbatim %}<twig:Alert/>{% endverbatim %}',
             '{% verbatim %}<twig:Alert/>{% endverbatim %}',
         ];
+
+        yield 'component_attr_spreading_self_closing' => [
+            '<twig:foobar bar="baz"{{...attr}}/>',
```

Comment:
Instead of having ... between {{ why not just do:
```html
<twig:foobar bar="baz" ...attr />
```

Reply:
For me eyes it's easier to find that it's a variable like in twig

Reply:
A little funky looking but what about `<twig:foobar bar="baz" :...attr />`?

Reply:
Looking at React:

```jsx
<button className={className} {...other} />
```

And Vue:

```
<button :className="className" v-bind="other">
```

Given these, the logical options are:

```
<twig:Button className={{ className }} {{...other}} />

or

<twig:Button live-bind="other">
```

And that `live-bind` thing looks silly :). So I agree with the proposed syntax.

Reply:
Note that JSX for Vue.js also use `{... }` syntax.

`{{... }}` is nice because the two brackets reminds us about Twig